packages/fiori-freestyle/test/__snapshots__/worklist.test.ts.snap

```diff
@@ -6929,12 +6929,12 @@ archive.zip
     \\"@sap/ux-ui5-fe-mockserver-middleware\\": \\"latest\\"
   },
   \\"scripts\\": {
-    \\"start\\": \\"fiori run --open 'test/flpSandbox.html#wrk1-tile'\\",
+    \\"start\\": \\"fiori run --open 'test/flpSandbox.html?sap-client=123#wrk1-tile'\\",
     \\"build\\": \\"ui5 build --config=ui5.yaml --clean-dest --dest dist\\",
     \\"deploy-config\\": \\"fiori add deploy-config\\",
-    \\"start-local\\": \\"fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html#wrk1-tile'\\",
-    \\"start-noflp\\": \\"fiori run --open 'index.html'\\",
-    \\"start-mock\\": \\"fiori run --config ui5-mock.yaml --open 'test/flpSandbox.html#wrk1-tile'\\"
+    \\"start-local\\": \\"fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html?sap-client=123#wrk1-tile'\\",
```

Comment:
Assume this sap-client is ignored for this app in preview as it is not a real ABAP backend.